Sources/Mustache/Library+FileSystem.swift

```diff
@@ -16,7 +16,7 @@ import Foundation
 
 extension MustacheLibrary {
     /// Load templates from a folder
-    static func loadTemplates(from directory: String, withExtension extension: String = "mustache") async throws -> [String: MustacheTemplate] {
+    static func loadTemplates(from directory: String, withExtension extension: String = "mustache") throws -> [String: MustacheTemplate] {
```

Comment:
> Library+FileSystem.swift:28:40: Instance method 'makeIterator' is unavailable from asynchronous contexts

`for case let path as String in enumerator` cannot be used in asynchronous contexts.
This function is not unnecessary to use asynchronous, so convert to synchronous function.

Reply:
Ah the enumerator stuff, I want to ditch it and replace with something else but in the meantime I guess we can remove the `async`.